constraint/pkg/client/regolib/src.go

```diff
@@ -4,6 +4,17 @@ const (
 	targetLibSrc = `
 package hooks["{{.Target}}"]
 
+violation[response] {
+	data.hooks["{{.Target}}"].library.autoreject_review[[rejection, constraint]]
+	review := get_default(input, "review", {})
+	response = {
```

Comment:
We should test that these results actually get passed on to the client. Unit tests & e2e tests.